packages/core/src/ObjectContainer.js

```diff
@@ -504,9 +518,50 @@ export default class ObjectContainer {
       }
     }
 
+    if (this._isSpread(name) && entry?.sharedInstance.length) {
```

Comment:
`entry?.sharedInstance` no need for the optional chaining.

Reply:
Same as above, entry can be null if not found.

Reply:
I think that better will be `Array.isArray(entry?.sharedInstance)` because object with length property throw Error for map function.

Reply:
What happens if I use spread operator for example for '...$Router' alias? I think it will works but It's not correct entry

Reply:
I added new error informing about invalid use of spread entry.